scss/components/_messages.scss

```diff
@@ -18,29 +23,30 @@
 .message {
   @include clearfix();
   background-size: 2rem;
-  background-color: rgba($ivory-dark, .5);
+  background-color: #eee6dd;
   border-color: $beige;
-  border-top: 2px solid $beige;
-  padding: 5rem 2rem 2rem 2rem;
+  border-style: solid;
+  border-width: 0 0 0 3px;
   margin: 2rem 0;
+  padding: 5rem 2rem 2rem 2rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `5rem 2rem 2rem` instead of `5rem 2rem 2rem 2rem`